src/appleseed.studio/mainwindow/rendering/lightpathstab.cpp

```diff
@@ -175,11 +175,15 @@ void LightPathsTab::create_light_paths_widget()
 {
     // Create the OpenGL widget.
     const CanvasProperties& props = m_project.get_frame()->image().properties();
+    QGLFormat format;
```

Comment:
`QGLFormat` isn only used by the widget directly, so you can create it in the constructor of `LightPathsWidget` and not here.

Reply:
It's needed to pass to the `QGLWidget` constructor so it needs to be present before the `LightPathsWidget` constructor in order to be used in the initialization list.

Reply:
Ok!

Reply:
With the way this works in qt5, we can indeed make the format in the constructor